### PR TITLE
Fix image display

### PR DESCRIPTION
**Pull request by BlueNealis**
Hi Kristy! 

I edited the image display, and made the profiles of us a little smaller.  

Let me know how it looks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Keep It Breezy
 
-![Success Baby Poster](https://turingschool.slack.com/files/U0327VCBG59/F03A8AFM4CR/screen_shot_2022-04-03_at_8.48.38_am.png "Success Baby Poster")
+![Success Baby Poster](readme-imgs/SuccessBaby.png)
 
 ## Table of Contents [create an anchor](#anchors-in-markdown)
```
Comment (kpn678) [inline README.md]: Can you remove the [create an anchor](#anchors-in-markdown)?

**Review by kpn678 — CHANGES_REQUESTED** [review]
I did mark one thing in the comments if you go look at it prior to merging!

File: README.md (hunk 1 above)
Comment (BlueNealis) [reply README.md]: Can do!

**Review by kpn678 — APPROVED** [review]
Yay, thank you for doing that!

**Pull request merged**